build/configs/rtl8720e/hello/Make.defs

```diff
@@ -54,11 +54,7 @@ include ${TOPDIR}/tools/Config.mk
 ARCH_FAMILY = $(patsubst "%",%,$(CONFIG_ARCH_FAMILY))
 include ${TOPDIR}/arch/$(CONFIG_ARCH)/src/$(ARCH_FAMILY)/Toolchain.defs
 
-ifeq ($(CONFIG_AMEBALITE_TRUSTZONE),y)
-LDSCRIPT = rlx8720e_img2_ns.ld
-else
-LDSCRIPT = rlx8720e_img2_is.ld
-endif
+LDSCRIPT = rlx8720e_img2.ld
```

Comment:
May I know why have we removed the other script and merged into one ?

Reply:
TrustZone and Non-TrustZone used the same layout for TP1e.,
So we should maintain only one script and remove the other.